src/widget-sources/widget-tileset-source.ts

```diff
@@ -106,9 +108,6 @@ export class WidgetTilesetSource extends WidgetSource<WidgetTilesetSourceProps>
     options,
   }: {
     spatialFilter: SpatialFilter;
-    // TODO(cleanup): As an optional property, 'uniqueIdProperty' will be easy to forget.
-    // Would it be better to configure it on the source function, rather than separately
-    // on the layer and in 'loadTiles()'?
```

Comment:
Removing this comment because `uniqueIdProperty` is inferred automatically in nearly all cases, the user shouldn't need to worry about it. In deck.gl this is a layer property, not a source/data property, and migrating that seems like too much of a breaking change for the benefit.